common/lib/xmodule/xmodule/modulestore/mongo/draft.py

```diff
@@ -651,7 +651,7 @@ def _internal(tier):
         if len(to_be_deleted) > 0:
             bulk_record = self._get_bulk_ops_record(root_usages[0].course_key)
             bulk_record.dirty = True
-            self.collection.remove({'_id': {'$in': to_be_deleted}}, safe=self.collection.safe)
+            self.collection.remove({'_id': {'$in': to_be_deleted}})
```

Comment:
Nit: `collection.remove` has been deprecated, can you change this to `delete_many`?